app/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+gradle.properties
```

Comment:
I don't think this should be ignored so that we can provide the set of keys that are necessary for the app to compile. We can then change the instructions to mention that they should provide their API key.

Reply:
Per our offline discussion, I've moved the API key to `app/secure.properties`, and also added a Toast in MainActivity when this variable isn't found to prompt the developer to add their API key - https://github.com/googlemaps/android-maps-ktx/pull/29/commits/27080920817f55f6cab5212839234b6569cdefe7. Project should build and tests pass even when the file or field isn't present.